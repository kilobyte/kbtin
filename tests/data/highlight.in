#sub ~ +
#ali x {#doh x {$0};#dos x {$x};#show {$x}}
#hi green foo
#hi red foo bar
#hi brown x*y
x x foo baz
x foo bar
x y x y x
