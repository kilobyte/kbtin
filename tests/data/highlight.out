x +2+foo+-1+ baz
+4+foo bar+-1+
y +6+x y+-1+ x
